src/librbd/cache/pwl/LogEntry.h

```diff
@@ -23,8 +23,8 @@ typedef std::list<std::shared_ptr<GenericWriteLogEntry>> GenericWriteLogEntries;
 
 class GenericLogEntry {
 public:
-  WriteLogPmemEntry ram_entry;
-  WriteLogPmemEntry *pmem_entry = nullptr;
+  WriteLogCacheEntry ram_entry;
+  WriteLogCacheEntry *pmem_entry = nullptr;
```

Comment:
Nit: still a reference to `pmem` in the base class

Reply:
Done